projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
Wow! Why to remove Netstandard?

Reply:
Due to most Span related APIs not available in it, and I don't think that we may need to support old .NET Framework versions.
An alternative solution would be to provide the Span API ourselves for `netstandard2.0` as a separate project (yes it would be unsafe), only compilable for `netstandard2.0`, so we won't have any unsafe on the main project, and that in `netcoreapp3.1` we would be using the standard provided APIs that should be safe and we can disable the unsafe in the main project.

Reply:
netstandard2.0 does not mean `old .NET Framework versions`.